LeftNavBarExample/LeftNavBarLibrary/build.gradle

```diff
@@ -3,7 +3,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.0.0'
+        classpath 'com.android.tools.build:gradle:2.2.1'
```

Comment:
Yes, I see your point.
Honestly I don't mind having this one as an independent project, but that would require a duplication of the gradle wrapper or can we avoid that somehow?